src/NuGetPackageVerifier/Program.cs

```diff
@@ -91,14 +91,16 @@ public static int Main(string[] args)
             }.ToDictionary(t => t.GetType().Name, t => t);
 
             var nupkgsPath = args[0];
-            var localPackageRepo = new LocalPackageRepository(
-                new DefaultPackagePathResolver(nupkgsPath),
-                new CustomPhysicalFileSystem(nupkgsPath));
-            var numPackagesInRepo = localPackageRepo.GetPackages().Count();
+            var dirInfo = new DirectoryInfo(nupkgsPath);
+            var nupkgs = dirInfo.GetFiles("*.nupkg", SearchOption.TopDirectoryOnly);
```

Comment:
The old code had logic in it to ignore `.symbols.nupkg` files - do we still need that?